File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
def solution(n, m):
    answer = [0, 0]
    
    minValue = min(n, m)
    
    for i in range(1, minValue + 1):
        if (n % i == 0 and m % i == 0):
            answer[0] = i
            answer[1] = n * m / i
    
    return answer

File: 프로그래머스/1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    // 최대 공약수 구하는 함수
    const getGCD = (num1, num2) => {
        let result = 1;
        for (let i = 1; i <= Math.min(num1, num2); i++) {
            if (num1 % i === 0 && num2 % i === 0) {
                result = i;
            }
        }
        return result;
    }
    
    const gdc = getGCD(n, m);
    return [gdc, n * m / gdc];
}